docs/src/whatsnew/latest.rst

```diff
@@ -104,6 +104,9 @@ This document explains the changes made to Iris for this release
    http://www.nationalarchives.gov.uk/doc/open-government-licence since this
    always works locally, but never within CI. (:pull:`4307`)
 
+#. `@wjbenfold`_ netCDF integration tests now skip TestConstrainedLoad if
```

Comment:
@wjbenfold We can't use `sphinx` to link to test classes, as they're not publicly documented as per the `iris` API, but perhaps put back ticks around `TestConstrainedLoad` to highlight it as a class?

Reply:
```suggestion
#. `@wjbenfold`_ netCDF integration tests now skip ``TestConstrainedLoad`` if
```